CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ The format is based on [Keep a Changelog]
 
 ## [Unreleased]
 
+- Payroll export start and end dates are the second Monday and Sunday in the
+  month
```

Comment:
Just a note that if the month starts on a Sunday, the second Sunday is before the second Monday.

Reply:
Yeah, it's not clear is it! I wasn't even sure something like this goes in the changelog? 

> Payroll export start date is the second Monday in the month with the end date the Sunday of the same week

How does that read?

Funny because I had a spec for a month starting on Sunday and took it out because I was worried about the spec looking so repetetive!

Reply:
Maybe:

> Payroll export start date is the second Monday in the month and the end date is the following Sunday